distributions/standalone/src/main/resources/MIGRATION.txt

```diff
@@ -8,6 +8,7 @@ Migration changes from 3.8.x to 3.9.x
 MULE-9931: Transaction log files size are now restricted by size, using a default size of 500 mb. The configured size is just an approximated value which may be exceeded based on the size of the record store by the transaction. This is configurable using the attribute queueTransactionFilesSize of the <configuration> element.
     The size restriction applies to the set of transaction log files for local transactions and XA transactions independently meaning that if both types of transactions are used for queue the set of tx files will use up to 1 gb.
 MULE-10100: Processing after a synchronous until successful resulting in a VoidMuleEvent will now continue with the original event.
+MULE-10306: Disable XML internal entity expansion by default because if allows DoS attacks. To restore previous behaviour use the expandInternalEntities="true" attribute in the xslt transformer.
```

Comment:
I'd recommend following the "is now" standard to explain the changes, something like: XML internal entity expansion is now disabled by default because...